app/src/main/res/values/strings.xml

```diff
@@ -40,4 +43,10 @@
     <string name="join_button_text">Join</string>
     <string name="join_fail_format">Failed to join lobby %d.</string>
     <string name="lobby_not_found">Lobby %d not found.</string>
+    <string name="add_a_playlist_manually">Add a playlist manually</string>
+    <string name="add_new_item">Add New Item</string>
```

Comment:
Have you forgotten to use it for a content description?

Reply:
Bien-vu, I forgot to remove it.